src/chocolatey.tests/infrastructure.app/nuget/NugetCommonSpecs.cs

```diff
@@ -179,5 +185,86 @@ public void Should_set_user_agent_string()
                 UserAgent.UserAgentString.Should().StartWith(expectedUserAgentString);
             }
         }
+
+        private class When_gets_package_dependencies : TinySpec
+        {
+            private Func<Task> _because;
+            private readonly Mock<SourceCacheContext> _sourceCacheContext = new Mock<SourceCacheContext>();
+            private readonly Mock<ILogger> _nugetLogger = new Mock<ILogger>();
+            private readonly List<NuGetEndpointResources> _nuGetEndpointResources = new List<NuGetEndpointResources>();
+            private readonly HashSet<SourcePackageDependencyInfo> _sourcePackageDependencyInfos = new HashSet<SourcePackageDependencyInfo>();
+            private readonly HashSet<PackageDependency> _packageDependencies = new HashSet<PackageDependency>();
+            private readonly Mock<SourceRepository> _sourceRepository = new Mock<SourceRepository>();
+            private readonly Mock<DependencyInfoResource> _dependencyInfoResource = new Mock<DependencyInfoResource>();
+            private PackageSource _packageSource;
+            private ChocolateyConfiguration _configuration;
+
+            public override void Context()
+            {
+                _configuration = new ChocolateyConfiguration();
+                _sourceCacheContext.ResetCalls();
+                _nugetLogger.ResetCalls();
+                _sourceRepository.ResetCalls();
+                _nuGetEndpointResources.Clear();
+                _sourcePackageDependencyInfos.Clear();
+                _packageDependencies.Clear();
+                _sourceRepository.Setup(r => r.GetResource<DependencyInfoResource>(It.IsAny<SourceCacheContext>())).Returns(_dependencyInfoResource.Object);
+                _packageSource = new PackageSource("C:\\packages");

```

Comment:
Does this setup an actual source at the location on disk? This should probably be an integration test if it's reaching out to the system.

I suspect based on the next line that it doesn't, and just needs a package source. We will want to verify this is the case though.

Reply:
I do not believe the code uses this location for file system access as part of the test.  The test passes with a different string like "fake-source", but that isn't much of a guarantee against IO.  Ideally I could provide a mock implementation of PackageSource, but that didn't seem to be viable.

Reply:
I'm good with saying this doesn't use the filesystem, I've run through the tests in Visual Studio and can confirm that this directory isn't being created or used.